### PR TITLE
Add twitter card meta-tags

### PR DESCRIPTION
**Pull request by hectoruch**
Task => https://www.pivotaltracker.com/story/show/130448201

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -25,6 +25,14 @@
   <meta name="theme-color" content="#18306d">
   <meta name="apple-mobile-web-app-capable" content="yes">
   <meta name="apple-mobile-web-app-status-bar-style" content="black">
+
+  <!-- META TAGS TWITTER CARD -->
+  <meta name="twitter:card" content="summary" />
+  <meta name="twitter:site" content="@globalfishwatch" />
+  <meta name="twitter:title" content="The First Global View of Commercial Fishing Activity" />
+  <meta name="twitter:description" content="Global Fishing Watch enables anyone with an Internet connection to see fishing activity anywhere in the ocean in near real-time — for free" />
+  <meta name="twitter:image" content="/image-twitter.png" />
```
Comment (tiagojsag) [inline app/index.html]: not working. probably needs to be the absolute URL

File: app/index.html [hunk 2]
```diff
@@ -25,6 +25,14 @@
   <meta name="theme-color" content="#18306d">
   <meta name="apple-mobile-web-app-capable" content="yes">
   <meta name="apple-mobile-web-app-status-bar-style" content="black">
+
+  <!-- META TAGS TWITTER CARD -->
+  <meta name="twitter:card" content="summary" />
+  <meta name="twitter:site" content="@globalfishwatch" />
+  <meta name="twitter:title" content="The First Global View of Commercial Fishing Activity" />
```
Comment (tiagojsag) [inline app/index.html]: switch to "Global Fishing Watch"

File: app/index.html [hunk 3]
```diff
@@ -25,6 +25,14 @@
   <meta name="theme-color" content="#18306d">
   <meta name="apple-mobile-web-app-capable" content="yes">
   <meta name="apple-mobile-web-app-status-bar-style" content="black">
+
+  <!-- META TAGS TWITTER CARD -->
+  <meta name="twitter:card" content="summary" />
+  <meta name="twitter:site" content="@globalfishwatch" />
+  <meta name="twitter:title" content="Global Fishing Watch" />
+  <meta name="twitter:description" content="Global Fishing Watch enables anyone with an Internet connection to see fishing activity anywhere in the ocean in near real-time — for free" />
+  <meta name="twitter:image" content="https://raw.githubusercontent.com/Vizzuality/GlobalFishingWatch/master/public/apple-touch-icon.png" />
```
Comment (tiagojsag) [inline app/index.html]: change this to the production url

**Comment by hectoruch** [comment]
Ready to review @tiagojsag

**Pull request merged**